runtime/client/ext-app-entry.js

```diff
@@ -52,6 +54,60 @@ function keepAlive (appId) {
   })
 }
 
+function stopAlive () {
+  clearInterval(aliveInterval)
+}
+
+function test (testGlobs) {
+  var tape = require('tape')
```

Comment:
Test framework has to be determined so that toolchains could be built on. The actual question is that there is no `tape` installation on release version.

Reply:
> The actual question is that there is no tape installation on release version.

Exactly.

Reply:
So the real problem is how to resolve side effects of adding `tape` to release builds.